hummingbot/model/inventory_cost.py

```diff
@@ -0,0 +1,75 @@
+from decimal import Decimal
+from typing import Optional
+
+from sqlalchemy import (
+    Column,
+    String,
+    Numeric,
+    Integer,
+    UniqueConstraint,
+)
+from sqlalchemy.orm import Session
+
+from . import HummingbotBase
+
+
+class InventoryCost(HummingbotBase):
+    __tablename__ = "InventoryCost"
```

Comment:
Do we actually need this new table in the sqlite? 
Since we already store trade fills, we can use that to calculate inventory cost, all we need is just a start time which can be config map variable (ie. when inventory_cost price type is set, it will also set inventory_cost_start_time).

Reply:
I'm not exactly remember why I choose to use separate table. Let's imagine user has some base asset amount which wasn't obtained via trading in the bot, how it should set the cost for this amount / how we can store it properly?

Reply:
> Let's imagine user has some base asset amount which wasn't obtained via trading in the bot, how it should set the cost for this amount / how we can store it properly?

@phbrgnomo suggested below in his feature request.

> Example:
First, the bot would have to set a price for the present inventory, and we have two options for that:
1 - The user defines the inventory cost starting price

Generally, I don't think we should store data which can be calculated/derived from existing data. This will create more maintenance down the road.

Reply:
@bitphage Sorry, you already store the inventory starting price when user already has the asset, can ignore my previous comment. Also, having the db table is fine for now, making changes to that logic now will be a lot of work.

Reply:
Ok. Generally, I agree that it would be better to automatically calculate inventory cost from trade fill history.

Reply:
I've also looked into `history` command and `calculate_performance_metrics`, not sure that using this solution to obtain average buy price is performance-perfect, as it's doing same calculations on the supplied trades every time.